group1/comm/static/comm_tool.css

```diff
@@ -444,22 +487,41 @@ ul.msgmenu li.red {
 
 ul.msgmenu li:hover {
   background-color: #e6e6e6;
-};
+}
 
 /* END MESSAGE */
 
+#player {
+    position:relative;
+    top:10px;
```

Comment:
Let's change these absolute widths to relative widths if possible, so that resizing the application does not lead to issues

Reply:
lines 496 and 497